problem_builder/public/js/mentoring_with_steps.js

```diff
@@ -111,7 +93,8 @@ function MentoringWithStepsBlock(runtime, element) {
                 // We are now showing the review step / end
                 // Update the number of attempts.
                 attemptsDOM.data('num_attempts', response.num_attempts);
-                updateGrade(response.grade_data);
+                reviewStepDOM.html($(response.review_html).html());
```

Comment:
@smarnach Yes, it removes the outermost HTML tag. It's included because the XBlock runtime puts it there and we can't control that. (I could bypass the normal block rendering pipeline which wraps it in that DIV but then we would miss other important steps in the render pipeline, such as link rewriting).

This could in theory be simplified to `reviewStepDOM.replaceWith($(response.review_html))` but that would invalidate the jQuery object / DOM reference `reviewStepDOM`, so I found this to be cleaner.